tests/test_utils.py

```diff
@@ -61,6 +61,11 @@ def test_get_image_online(self, image_id, tmpdir, mocker):
         assert isinstance(img, np.ndarray)
         assert img.dtype == np.uint8
 
+    def test_get_image_corrupted(self, tmpdir):
```

Comment:
So it seems like we have a `@pytest.mark.internet` mark for tests that require internet. Maybe we can use it here too?

What is a bit tricky about it is that by default it is going to be excluded since our config looks like this (taken from `tox.ini`)

```ini
[pytest]
testpaths = tests
addopts =
    -m "not todo and not slow and not internet"
    --cov
    --cov-config=tox.ini
    --no-cov-on-fail
    --durations=20
    --verbosity=1
    --color=yes
markers =
    internet: requires connection to the internet
    slow: mark denoting a test that is too slow
    todo: mark denoting a test that is not written yet

```


So by default it would not be run in the CI. I don't know. What do you think is the best approach?

Reply:
We can go for that solution! :) I can imagine that we are going to continue testing locally all the tests!